File: INI2/hypotenuse.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func main() {
	if len(os.Args) < 3 {
		fmt.Printf("Usage: %v <num1> <num2>\n",
		os.Args[0])
		os.Exit(1)
	} else {
		num1, err := strconv.Atoi(os.Args[1])
		if err != nil {
			fmt.Println("error:", err)
		}
		num2, err := strconv.Atoi(os.Args[2])
		if err != nil {
			fmt.Println("error:", err)
		}
		ans := num1*num1 + num2*num2
		fmt.Println(ans)
	}
}

File: INI2/hypotenuse.c
#include <stdio.h>

int main(int argc, char **argv) {
    if (argc < 3) {
        printf("Usage: %s <num1> <num2>\n",
                argv[0]);
        return 1;
    } else {
        int num1 = atoi(argv[1]);
        int num2 = atoi(argv[2]);
        int ans = num1*num1 + num2*num2;
        printf("%d\n", ans);
    }
    return 0;
}
